lib/matplotlib/backends/_backend_tk.py

```diff
@@ -44,6 +44,28 @@ def _restore_foreground_window_at_end():
             _c_internal_utils.Win32_SetForegroundWindow(foreground)
 
 
+_blit_args = {}
+# Initialize to a non-empty string that is not a Tcl command
+_blit_tcl_name = "29345091836409813"
```

Comment:
why not directly initialize this to the final name here? e.g. `"mpl_blit_" + uuid.uuid4().hex`

Reply:
I love this idea, I always thought the initial name was tacky but for some reason I never came up with this fix!

Reply:
Just a quick question, didn't we have a bug report come by recently because we were using one of the uuid functions, which was using a "weak" shasum algorithm, so their security-restricted system couldn't compute it?

Reply:
In a pinch we could hardcode a uuid, they are universally unique after all...